src/main/java/core/basesyntax/strategy/impl/SupplyOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperationImpl implements OperationHandler {
+    @Override
+    public void applyOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() <= MIN_QUANTITY) {
```

Comment:
The constant `MIN_QUANTITY` is not defined within this class or imported from elsewhere. You should define this constant in this class or import it if it's located in another class.